gravitee-apim-console-webui/src/entities/management-api-v2/api/v4/connectorVM.ts

```diff
@@ -25,6 +25,7 @@ export type ConnectorVM = {
    */
   isEnterprise: boolean;
   supportedListenerType: ListenerType;
+  supportedQos: Qos[];
```

Comment:
This class is used also for endpoint lists. Is QoS pertinent for endpoints?

Reply:
Nope, endpoints have a list of supported Qos, but you can only configure en Qos on the entrypoint.
the gateway will try to determine the best "match" between what has been asked and what is possible.
https://gravitee.slab.com/posts/conceptual-changes-aakyswte#hs27v-compatibilities